arkane/pdep.py

```diff
@@ -641,9 +641,11 @@ def draw(self, output_directory, file_format='pdf'):
 
         NetworkDrawer().draw(self.network, file_format=file_format, path=path)
 
-    def save_input_file(self, path):
+    def save_input_file(self, path, use_adj_list=False):
```

Comment:
Is there a reason we need to have this be an option. What's the downside to just making it always be an adjlist?